.vscode/settings.json

```diff
@@ -24,5 +24,12 @@
 	},
 	"[scss]": {
 		"editor.rulers": [80]
-	}
+	},
+	"Lua.diagnostics.globals": [
```

Comment:
The pro way to do this is:

1. Disable diagnostics in your Lua extension
2. Setup a [selene](https://github.com/Kampfkarren/selene) standard library, and use that as your linter.